TankControl.ino

```diff
@@ -413,6 +433,20 @@ void loop() {
     s2 = !digitalRead(SENSOR2);
     s3 = !digitalRead(SENSOR3);
 
+    if(((!s3prev && !s2prev && !s1prev) && (!s3 && s2 && !s1)) || ((!s3prev && !s2prev) || (!s3 && s2))) {
```

Comment:
((!s3prev && !s2prev && !s1prev) && (!s3 && !s2 && s1))

Reply:
Switch debouncing from sensor to be considered.

Reply:
Cannot be implemented because of hysteresis (sensor) issue.